src/icons/Activity.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import colors from 'src/styles/colors'
+import Svg, { Path } from 'svgs'
+
+interface Props {
+  size?: number
```

Comment:
Do we plan to use `size`?

Reply:
oops, no lol classic copy and paste and didn't check 😅 i'll update